jetty-util/src/main/java/org/eclipse/jetty/util/resource/Resource.java

```diff
@@ -921,4 +913,97 @@ public static URL toURL(File file) throws MalformedURLException
     {
         return file.toURI().toURL();
     }
+
+    /**
+     * Parse a list of String delimited resources and
+     * return the List of Resources instances it represents.
+     * <p>
+     * Supports glob references that end in {@code /*} or {@code \*}.
+     * Glob references will only iterate through the level specified and will not traverse
+     * found directories within the glob reference.
+     * </p>
+     *
+     * @param delimitedReferences the comma {@code ,} or semicolon {@code ;} delimited
+     * String of resource references.
+     * @param globDirs true if glob references return directories within the glob as well
```

Comment:
Isn't this the opposite of the foregoing comment: "Glob references will only iterate through the level specified and will not traverse found directories within the glob reference"? The intention is that when the wildcard char * is used, deep directory traversal is not supported.